src/components/Resulttablerows.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Stayindays from './Stayindays';
+const Resulttablerows = (props) => {
```

Comment:
When naming you're components, it would be better to use upper case letters for different words, so this would become `ResultTableRow`